lib/quality/which.rb

```diff
@@ -0,0 +1,14 @@
+class Which
+  # http://stackoverflow.com/questions/2108727/
+  #   which-in-ruby-checking-if-program-exists-in-path-from-ruby
+  def which(cmd)
+    exts = ENV['PATHEXT'] ? ENV['PATHEXT'].split(';') : ['']
+    ENV['PATH'].split(File::PATH_SEPARATOR).each do |path|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.